modules/internal/String.chpl

```diff
@@ -615,10 +786,11 @@ module String {
       return localThis[start..end];
     }
 
-    inline proc strip(leading=true, trailing=true) : string {
-      return this.strip(" \t\r\n", leading, trailing);
-    }
-
+    /*
+      Splits the string on `sep` into a `3*string` consisting of the section
```

Comment:
My personal preference would be to call it a 3 string tuple instead of using the syntax, but you don't have to change it.